src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -56,12 +56,7 @@ public AddCommand parse(String args) throws ParseException {
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
         // add gender in parser
-        Gender gender;
-        if (!arePrefixesPresent(argMultimap, PREFIX_GENDER)) {
-            gender = Gender.getNoGender();
-        } else {
-            gender = ParserUtil.parseGender(argMultimap.getValue(PREFIX_GENDER).get(), Boolean.FALSE);
-        }
+        Gender gender = ParserUtil.parseGender(argMultimap.getValue(PREFIX_GENDER).get());
```

Comment:
I feel that moving this statement up to be between `address` and `tag` would be better, as it would follow the ordering that is used throughout